.env

```diff
@@ -97,7 +97,9 @@ DEVTOOLSET_VERSION=
 # actions cache up to date for the macOS wheels:
 #   https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml
 # vcpkg minimum version "09adfdc8cdad76345b7cc7f3305899e1cbd66297" due to CVE-2022-3786
-VCPKG="2871ddd918cecb9cb642bcb9c56897f397283192"
+# VCPKG="a7b6122f6b6504d16d96117336a0562693579933"  # 2023.02.24 release version
+# TODO(sjperkins): Change to an official release version
+VCPKG="b619a233fbf7b2c9765fb4458f3ecb05bd3166e3"    # 2023.04.03
```

Comment:
The above isn't a released version (but does contain the openssl linux-headers fixes). Would it make sense to wait for a new release? vcpkg seems to release every 2 months.

Reply:
Could you open a new issue to update a release version and update the TODO comment to refer the new issue?

Reply:
Yes, have done so in https://github.com/apache/arrow/issues/35099